### PR TITLE
Upgrade Jupyter apis to v1.6.0

### PR DESCRIPTION
**Pull request by mathis-marcotte**
This branch was built on top of "feat-upgrade-1.4"

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 coverage_unit.txt
 jupyter-apis
+meow-old
+crud-web-apps
```
Comment (wg102) [inline .gitignore:4]: If not needed, please remove

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - feat-upgrade-1.4
```
Comment (wg102) [inline .github/workflows/publish.yml:6]: If not needed please remove

File: .gitignore (hunk 1 above)
Comment (Jose-Matsuda) [reply .gitignore:4]: no the meow is necessary

Comment (mathis-marcotte) [reply .gitignore]: @Jose-Matsuda Is it really or do you just not want the meow to go away?

**Review by Jose-Matsuda — APPROVED** [review]
LGTM, will test and review on dev

**Pull request merged**